misc/config_tools/board_config/board_cfg_gen.py

```diff
@@ -45,7 +45,8 @@ def main(args):
     common.get_vm_types()
 
     if common.VM_COUNT > common.MAX_VM_NUM:
-        err_dic['vm count'] = "The vm count in config xml should be less or equal {}!".format(common.MAX_VM_NUM)
+        err_dic['vm count'] = "The vm count in config xml should be less or equal hv/CAPACITIES/MAX_VM_NUM !" \
```

Comment:
Not sure what you mean by "vm count", could this be replaced by "number of VMs"?

```suggestion
        err_dic['vm count'] = "The number of VMs defined in the scenario XML should be less or equal than hv.CAPACITIES.MAX_VM_NUM. This number is currently set to {}!".format(common.MAX_VM_NUM)
```